packages/common/src/types/schema.ts

```diff
@@ -419,6 +414,12 @@ export type DeleteSupplierRequisitionResponseWithId = {
   response?: Maybe<DeleteSupplierRequisitionResponse>;
 };
 
+export type EqualFilterBigNumberInput = {
+  equalAny?: InputMaybe<Array<Scalars['Int']>>;
+  equalTo?: InputMaybe<Scalars['Int']>;
+  notEqualTo?: InputMaybe<Scalars['Int']>;
+};
```

Comment:
Are we not allowed to use little numbers? So many limitations of this api 😱

Reply:
😄  I missed that! Is `Int` a 'BigNumber' then?